pkgs/development/python-modules/yq/default.nix

```
@@ -14,27 +15,31 @@
 
 buildPythonApplication rec {
```

```suggestion
buildPythonPackage rec {
```
or else you won't be able to import it